docs/learn-treasury.md

```diff
@@ -103,7 +103,7 @@ values: 20 KSM or 5% of the requested amount.
 Once created, your proposal will become visible in the Treasury screen and the council can start
 voting on it.
 
-![A pending proposal](/img/treasury/proposal.jpg)
+![A pending proposal](/img/treasury/new-proposal.jpg)
```

Comment:
This image doesn't exactly match what the text above is describing it as. I'm thinking for users, it would make most sense to place an image that has a proposal in the treasury rather than the page itself.